Source/WebKit/MigratedHeaders-input.xcfilelist

```diff
@@ -0,0 +1,281 @@
+# This is a list of every header WebKit "re-exports" from WebCore or
+# WebKitLegacy, to correspond with reexported symbols. Because some headers are
+# differently public or private based on the target platform, these headers are
+# copied via the "Migrate WebCore and WebKitLegacy Headers" script phase
+# instead of the native Headers phase. 
+
+# The script matches each line of input to the corresponding line in
+# MigratedHeaders-output.xcfilelist, so the order of file names must match.
```

Comment:
Can the script verify that file names do match?

Reply:
Good idea!